Jenkinsfile

```diff
@@ -20,7 +20,9 @@ pipeline {
 						-Pbuild-individual-bundles -Pbree-libs -Papi-check \
 						-DskipTests=false -Dcompare-version-with-baselines.skip=false \
 						-Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true \
-						-Dmaven.compiler.failOnWarning=true -Dproject.build.sourceEncoding=UTF-8 
+						-Dmaven.compiler.failOnWarning=true -Dproject.build.sourceEncoding=UTF-8 \
```

Comment:
I believe what you want is `timestamps()` above like in https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/0beaf7b6ffe8df199342b2f0644d555e43367f34/Jenkinsfile#L5

Reply:
intersting: now i got:
`11:53:50  OS name: "linux", version: "5.14.14-200.fc34.x86_64", arch: "amd64", family: "unix"
17:53:58  15:53:58,075 [INFO] Scanning for projects...`

i.e. three different times. 11,15,17 o'clock. 
that timestamps() adds clock from jenkins,
while org.slf4j.simpleLogger.showDateTime=true adds time from surefire. I wonder where 11 o'clock is comming from

Reply:
https://ci.eclipse.org/platform/job/eclipse.platform.runtime/job/PR-20/
says it 11 o'clock - time of the jenkins server? A global unique time would be to easy.

Reply:
i dislike jenkins timestamps() since i adds the  time in front of every line. for example:
 "17:57:43  org.eclipse.core.internal.expressions.tests.PropertyTesterTests.testDynamicPlugin  Time elapsed: 0 s"
which is misleading since that is the time when the result was read and not the time when the test did run.